custom_components/e3dc_rscp/e3dc_proxy.py

```diff
@@ -273,7 +278,13 @@ def get_timezone(self) -> str:
     @e3dc_call
     def poll(self) -> dict[str, Any]:
         """Poll E3DC current state."""
-        return self.e3dc.poll(keepAlive=True)
+        result : dict[str, Any] = self.e3dc.poll(keepAlive=True)
+
+        result["power"] = {
```

Comment:
Please move this to coordinator.py instead, the proxy functions should all encapsulate exactly one E3DC call. I introduced this class to work around missing code in pye3dc, which is why I want that proxy streamlined.
Move this into _load_and_process_powersettings in the coordinator (watch for exception handling) or even - if you want - implement your own _load... call there. I would do the former, just add your poller into the existing try block and then update _mydata accordingly. Makes things a bit cleaner.

Reply:
@volkerrichert Thanks for all the other changes, can you move this one to the coordinator file? This is the only thing I'd like to have for better maintainability.
If you don't have the time, let me know, I might have some time next week to move it.
Besides this detail, consider the PR approved in its intent.